src/definition.ts

```diff
@@ -61,6 +70,17 @@ export const structurize = (fhirDefinition: {
       res[property] = fhirDefinition[property]
     }
 
+    // Build the metadata structure on the definition object
+    metaProperties.forEach(fill)
+
+    // Add the metadata extension fields in case the definition is an extension.
+    if (
+      fhirDefinition.type === 'Extension' &&
+      fhirDefinition.derivation === 'constraint'
```

Comment:
I don't remember if we use this a lot but do you think it would be worth having functions isExtension, isProfile, ...?

Reply:
it would be worth if we implemented our own class StructureDefinition (we would have an accessor isExtension for instance) but since we're dealing with simple JSON objects I don't think it's worth it **right now** (maybe later)

Reply:
Fine by me!